### PR TITLE
DOCSP-31489 Count Result Documents

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION
Create an "Count Pipeline Result Documents" task page. Source material from [here](https://www.mongodb.com/docs/compass/current/aggregation-pipeline-builder/#refresh-document-counts).

## STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-31489-refresh-document-counts/agg-pipeline-builder/count-pipeline-results/

## JIRA
https://jira.mongodb.org/browse/DOCSP-31489

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64e3a60b57e37806c5fb355c

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/agg-pipeline-builder/count-pipeline-results.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+   :depth: 1
+   :class: singlecol
+
+You can view the number of documents outputted by your pipeline with the 
+:guilabel:`count results` button.
+
+About this Task 
+---------------
+
+When you delete or add a document, the :guilabel:`count results` value on the
```
Comment (jmd-mongo) [inline source/agg-pipeline-builder/count-pipeline-results.txt:22]: [suggestion]

Considering rephrasing this as an imperative. Spitballing:

```
When you delete or add a document, you must manually refresh the 
:guilabel:`count results` value on the :guilabel:`Aggregations` tab to 
reflect the new document count.
```

File: source/agg-pipeline-builder/count-pipeline-results.txt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Steps
+-----
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: Click the count results button
+
+      After you run your pipeline, click :guilabel:`count results`, which 
+      appears under the :guilabel:`Run` button. The button will update with the
```
Comment (jmd-mongo) [inline source/agg-pipeline-builder/count-pipeline-results.txt:43]: [question]

Could we make "The button will update with the count..." more specific? Is this saying the Run button will updated with count? I'm a little confused. 

Thank you!

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the updates and improvements, @ajhuh-mdb. I've left a question and a suggestion for your consideration.

Thanks!
Joe

File: source/agg-pipeline-builder/count-pipeline-results.txt (hunk 1 above)
Comment (ajhuh-mdb) [reply source/agg-pipeline-builder/count-pipeline-results.txt:22]: Updated!

File: source/agg-pipeline-builder/count-pipeline-results.txt (hunk 2 above)
Comment (ajhuh-mdb) [reply source/agg-pipeline-builder/count-pipeline-results.txt:43]: Definitely! Updated.

**Comment by ajhuh-mdb** [comment]
Thanks for taking a look @jmd-mongo! This should be ready for another look.

**Review by jmd-mongo — APPROVED** [review]
Thanks for the updates, @ajhuh-mdb! This one LGTM at this point.

Thanks!
Joe

**Pull request merged**